Formula/kompose.rb

```diff
@@ -0,0 +1,23 @@
+class Compose2kube < Formula
```

Comment:
The class and filename are not in sync: https://bot.brew.sh/job/Homebrew%20Core%20Pull%20Requests/8340/version=el_capitan/console